### ISSUE TITLE
Tried making Fish Soup

### ISSUE THREAD

**Issue by MrWigggles** [CleverRaven/Cataclysm-DDA#44559]
DEBUG    : Attempted a recipe with no available components!

 FUNCTION : comp_selection<item_comp> player::select_item_component(const std::vector<item_comp>&, int, inventory&, bool, const std::function<bool(const item&)>&, bool)
 FILE     : src/crafting.cpp
 LINE     : 1529

**Comment by wapcaplet** [comment]
I could not reproduce this error when cratfing fish soup in 0.E-6411-gf08120bad9. I was able to craft fish soup successfully using a mess kit, pot, eggs, and dried fish:

![image](https://user-images.githubusercontent.com/238395/94959870-287cfd80-04af-11eb-907b-a70ced337105.png)

@MrWigggles Can you please provide your game version number, and the steps you followed to reproduce the error?

**Comment by MrWigggles** [comment]
Stable 0.E-2
Uh. Steps to reproduce.
Loosely.
Make broth. Have it in an adjacent tile.
Have anything being wielded. I think for this in particular it was a weapon
or nother hot food item.
I was using a Hobo Stove inside a building, with wood ash inside of it and
it was either burning a few planks or a log.
I made it with freshly butchered fish. Freshly made broth. I dont remember
the other ingredients I used.
When I was trying to start cooking it from the Craft menu it ask you what
to do with the held item.
I press escaped. Then dealt with the wielded object by putting it in my
inventory.
Then I went to the inventory with (i) and didnt wield, and just Activated
it.
